Makefile.toml

```
@@ -1352,5 +1355,56 @@ rm -rf ${BUILDSYS_STATE_DIR}
 '''
 ]
 
+[tasks.install-testsys]
```

I think I prefer keeping the cargo make task names agnostic about the underlying tool/implementation. 

Similar to how we have `tasks.build-tools` instead of `tasks.install-buildsys` and `tasks.publish-tools` instead of `tasks.install-pubsys`.

```suggestion
[tasks.test-tools]
```

If we want to further differentiate from `unit-test`, then maybe `integ-test-tools` would make more sense.